doc/setup_instructions.md

```diff
@@ -21,7 +21,7 @@ Thank you!
 Prerequisites
 =============
 
-Godot mono version
+Godot .Net version
```

Comment:
```suggestion
Godot .NET version
```

.NET is written uppercase (on the Godot website) so this is more consistent with the wording.